src/AltinnCore/Designer/Repository/ReleaseDbRepository.cs

```diff
@@ -0,0 +1,23 @@
+using AltinnCore.Designer.Infrastructure.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace AltinnCore.Designer.Repository
+{
+    /// <summary>
+    /// ReleaseDbRepository
+    /// </summary>
+    public class ReleaseDbRepository : DocumentDbRepository
```

Comment:
In Platform we've used the term {model}Repository. Could that be used in designer as well ?

Reply:
Agreed, done